.github/ISSUE_TEMPLATE/Bug-Report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: textarea
+    id: what-was-expected
+    attributes:
+      label: What did you expect to happen?
+      placeholder: Tell us what you wanted!
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: How can we reproduce this?
+      description: Please provide as much detail as possible so that we may try to reproduce and debug.
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: textarea
+    id: os
+    attributes:
+      label: Operating System and Version
+      description: Tell us about your development environment
+      placeholder: "Example: Ubuntu 20.04"
+      validations:
+        required: true
+  - type: textarea
+    id: os
+    attributes:
+      label: Execution Environment
+      description: How are you running tinkerbell?
+      placeholder: "Example: Vagrant libvirt sandbox"
+      valiations:
+        required: true
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our [Code of Conduct](./CODE_OF_CONDUCT.md)
```

Comment:
Can you absolute link to the COC in `tinkerbell/org`? I think this relative link won't work for other repos that get this template

Reply:
Merged and updated!